diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
index 6748c93..d1be23b 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
@@ -85,10 +85,6 @@ import {
 } from './constants';
 import type { AnimateToPositionType, BottomSheetProps } from './types';
 
-Animated.addWhitelistedUIProps({
-  decelerationRate: true,
-});
-
 type BottomSheet = BottomSheetMethods;
 
 const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
@@ -1746,6 +1742,23 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
       [_providedAnimatedPosition, topInset]
     );
 
+    /** Reset position of Bottom Sheet during screen rotate */
+    useAnimatedReaction(
+      () => animatedLayoutState.value.containerHeight,
+      (newHeight, previousHeight) => {
+        if (newHeight === previousHeight) {
+          return;
+        }
+        if (!previousHeight || animatedPosition.value < previousHeight) {
+          // Do not reset position while bottom sheet is open
+          return;
+        }
+
+        animatedPosition.value = newHeight;
+      },
+      [animatedLayoutState]
+    );
+
     /**
      * sets provided animated index
      */
diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetDebugView/ReText.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetDebugView/ReText.tsx
index 4c998e3..f64ad6a 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetDebugView/ReText.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetDebugView/ReText.tsx
@@ -15,8 +15,6 @@ interface TextProps {
 
 const AnimatedTextInput = Animated.createAnimatedComponent(TextInput);
 
-Animated.addWhitelistedNativeProps({ text: true });
-
 const ReText = ({ text, value: _providedValue, style }: TextProps) => {
   const providedValue = useDerivedValue(() => {
     if (!_providedValue) {
